### ISSUE TITLE
feat: WWE Network region lock US

### ISSUE THREAD

**Issue by SHAD0WBULLET707** [ReVanced/revanced-patches#3464]
### Feature description

Unlocks WWE Network in the US without the aid of a VPN.
https://www.apkmirror.com/apk/wwe-inc/wwe/
Don't know if the feature would be a location spoofer or region unlock

### Motivation

Don't have to use a VPN to access WWE Network in the US. Since its accessed elsewhere outside the US.
The app will sunset until the 2025 Netflix and would be nice to see what you guys can do.
No word from WWE will kill the app entirely and keep it as a archival.

### Acknowledgements

- [X] I have checked all open and closed feature requests and this is not a duplicate
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by SHAD0WBULLET707** [comment]
> If it's checking for location then it almost surely is using IP address, which means a VPN is the best that can be done.
> 
> If it is using the device location (seems unlikely since it would require asking for permission), then any mock location app can be used.

![WWE Ntwork](https://github.com/user-attachments/assets/b7e325d5-798f-4555-a42a-9735bfb1c46c)
[manifest.json](https://github.com/user-attachments/files/16326170/manifest.json)
https://addons.mozilla.org/en-US/firefox/addon/wwe-unblocker/

It could be possible, this browser extensions shows how to do it. The old browser extension used to block it via location spoofing and no VPN block on 2.5 version of the Network.
It shows hat can be done in the manifest that could make it work on Android app.

**Comment by oSumAtrIX** [comment]
The extension expects a VPN.

**Comment by SHAD0WBULLET707** [comment]
> The extension expects a VPN.

Yeah and could work and bypass the VPN restrictions, cause WWE usually blocks some VPN servers.
So other VPN servers can work properly. Same popup comes up on the android app too if its the right VPN that WWE doesn't block then it works.
![297406](https://github.com/user-attachments/assets/7495a0bd-cfe7-45a7-8647-3cc7e9a73498)

**Comment by oSumAtrIX** [comment]
You initially requested

> Unlocks WWE Network in the US without the aid of a VPN.

Refer to https://github.com/ReVanced/revanced-patches/issues/3464#issuecomment-2241780550

**Comment by SHAD0WBULLET707** [comment]
> You initially requested
> 
> > Unlocks WWE Network in the US without the aid of a VPN.
> 
> Refer to [#3464 (comment)](https://github.com/ReVanced/revanced-patches/issues/3464#issuecomment-2241780550)

should I edit the request or create a new one?

**Comment by oSumAtrIX** [comment]
No, you can use a VPN.

**Comment by SHAD0WBULLET707** [comment]
> No, you can use a good VPN
> 
> > if its the right VPN that WWE doesn't block then it works.

Guess this wont move forward? Stick with a VPN instead?
